articles/concepts/toc.yml

```diff
@@ -16,3 +16,7 @@
   href: circuits.md
 - name: Quantum oracles
   href: oracles.md
+- name: Further reading
```

Comment:
I'm not sure I agree with further reading being under the "theoretical concepts" section? We may want to instead broaden the list of books, courses, and articles away from being a list of arXiv papers, and promote the section more? https://github.com/ebraminio/awesome-qsharp and https://qsharp.community/ are both good places to start looking for broader resources. @tcNickolas, are there any you'd like to suggest here as well?

Reply:
I'd like to bump this, as I want to make sure that we don't bury further reading references.